src/opera/util/render_jinja2.py

```diff
@@ -202,6 +204,132 @@ def python_type_to_xml_type(obj) -> str:
     return XML_TYPES[obj]
 
 
+def augment_measured_parameters(measured_parameters: dict, mpc_path: str, logger: PgeLogger) -> dict:
+    """
+    Augment the measured parameters dict of GeoTIFF metadata into a dict of dicts
+    containing the needed fields for the MeasuredParameters section of the ISO XML
+    file.
+
+    For HDF5 or NetCDF metadata, use augment_hd5_measured_parameters()
+
+    Parameters
+    ----------
+    measured_parameters : dict
+       The GeoTIFF metadata from the output product. See get_geotiff_metadata()
+    mpc_path: str
+        Path to the Measured Parameters Descriptions YAML file
```

Comment:
Looking over how this function works again, and I think we need to document the following:

This function can accept `None` for `mpc_path`, in which case all descriptions will be filled in with `Not Provided`. Otherwise, any missing descriptions are filled in with `!Not Found!`, indicating a configuration error.